### PR TITLE
Monitor fixes

### PR DESCRIPTION
**Pull request by Ealdwulf**
This PR:
 - causes terminate signals to be passed to the child process from the monitor
 - causes the monitor to not launch, just exec the child process, when MONITOR_PID is already set (IE, a monitor is already running)
 For OS-313
Note that while this passes its unit test, I haven't tried it in the updater yet.

### CODE REVIEW

File: tests/fixtures/environ.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+#
+# monitor.py
```
Comment (radujipa) [inline tests/fixtures/environ.py]: *environ.py

File: tests/fixtures/environ.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+#
+# Copyright (C) 2018 Kano Computing Ltd.
+# License: http://www.gnu.org/licenses/gpl-2.0.txt GNU GPL v2
+#
+# Fixtures for faking internet status
```
Comment (radujipa) [inline tests/fixtures/environ.py]: copy/paste

File: tests/monitor_wrap.py [hunk 3]
```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env python
+#
+# monitor_wrap.py
+#
+# Copyright (C) 2018 Kano Computing Ltd.
+# License: http://www.gnu.org/licenses/gpl-2.0.txt GNU GPL v2x
+#
+# Wrapper for when the monitor needs to be tested in a separate process to pytesta
```
Comment (radujipa) [inline tests/monitor_wrap.py]: *pytest

**Comment by radujipa** [comment]
There are a few of these also:

```
----------------------------- Captured stderr call -----------------------------
Traceback (most recent call last):
  File "./tests/monitor_wrap.py", line 11, in <module>
    import kano_updater.monitor
  File "/home/kanux/jenkins-agent/workspace/_kano-updater_monitor_fixes-MIC56AVCM5P647TJFISVZMO6RL6Z4N5FUTATY5DVY2NKVS3C32AQ@3/kano_updater/monitor.py", line 17, in <module>
    import kano_updater.utils
  File "/home/kanux/jenkins-agent/workspace/_kano-updater_monitor_fixes-MIC56AVCM5P647TJFISVZMO6RL6Z4N5FUTATY5DVY2NKVS3C32AQ@3/kano_updater/utils.py", line 38, in <module>
    import kano_updater.apt_wrapper
  File "/home/kanux/jenkins-agent/workspace/_kano-updater_monitor_fixes-MIC56AVCM5P647TJFISVZMO6RL6Z4N5FUTATY5DVY2NKVS3C32AQ@3/kano_updater/apt_wrapper.py", line 9, in <module>
    import apt
ImportError: No module named apt
```

**Comment by Ealdwulf** [comment]
Thanks @radujipa Fixed now.

**Pull request merged**